src/Symfony/Component/HttpClient/Tests/HttpOptionsTest.php

```diff
@@ -30,7 +30,7 @@ public function provideSetAuthBasic(): iterable
     /**
      * @dataProvider provideSetAuthBasic
      */
-    public function testSetAuthBasic(string $expected, string $user, string $password = '')
+    public function testSetAuthBasic(string $expected, string $user, #[\SensitiveParameter] string $password = '')
```

Comment:
Do we really need to add the attribute in tests?

Reply:
Clearly no.